### PR TITLE
Update search tests

### PR DESCRIPTION
**Pull request by m8ttyB**
Addresses issue #385:

* update locators to work with the latest super search page changes
* update method names in Super Search POM to be more specific - hopefully will help in future archeological digs in the code base

### CODE REVIEW

File: pages/super_search_page.py [hunk 1]
```diff
@@ -15,27 +15,29 @@ class CrashStatsSuperSearch(CrashStatsBasePage):
     _page_title = 'Search - Mozilla Crash Reports'
 
     # Search parameters section
-    _field_text_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] > div:nth-child(2) span')
+    _facet_text_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] > div:nth-child(2) span:first-child')
+    _facet_field_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] .select2-container.field')
+    _facet_field_dropdown = (By.CSS_SELECTOR, 'fieldset[id="%s"] .select2-container.field')
```
Comment (stephendonner) [inline pages/super_search_page.py]: With only my simple grepping, this appears unused, and identical to _facet_field_locator, which is used (instead?)

**Comment by stephendonner** [comment]
The changes I understand look good to me, and this passes well against both staging and prod, but it might be worth getting another r? for good measure, from someone who's more familiar with our automation in general/locators, etc. r+ with that caveat.

(Also, that unused, duplicated locator I pointed out.)

File: pages/super_search_page.py (hunk 1 above)
Comment (m8ttyB) [inline pages/super_search_page.py]: Just testing you ;-) no, seriously, good catch. That is detritus left over from an earlier experiment.

**Comment by justinpotts** [comment]
r+ 

euro4

**Comment by m8ttyB** [comment]
Gracias -- doing a wee bit of squashing and then mergification.

**Pull request merged**